index.html

```diff
@@ -94,6 +94,115 @@ <h2>
     <section id="conformance">
       <!-- This section is filled automatically by ReSpec. -->
     </section>
+
+    <section>
+      <h2>
+        Design Constraints and Motivation
+      </h2>
+
+      <p>
+        This DID method was developed with the following design constraints:
+      </p>
+
+      <ol>
+        <li><strong>Backwards compatibility</strong>:  Resolvable by <code>did:web</code>
+          resolvers (for a given <code>did:webhash</code> DID, if the
+          <code>webhash</code> method keyword is changed to <code>web</code>,
+          an existing DID Web resolver is able to fetch and process it).
+        </li>
+        <li><strong>Static web hosting</strong>: DID Documents must be able to
+          be hosted on static web servers (with no support for query parameters
+          or custom HTTP headers).</li>
```

Comment:
```suggestion
          be hosted conformantly on static web servers (although optional features like 
          for query parameters or custom HTTP headers may require dynamic service).</li>
```

Reply:
The 'no query parameters' is on purpose. The core design requirement is that `did:webhash` should be able to be hosted by _static_ web servers (like Github pages). So, query parameters are out.